app/src/main/java/cm/aptoide/pt/editorial/EditorialManager.java

```diff
@@ -72,7 +72,7 @@ public Completable downloadApp(EditorialDownloadEvent editorialDownloadEvent) {
         editorialDownloadEvent.getMd5(), editorialDownloadEvent.getIcon(),
         editorialDownloadEvent.getVerName(), editorialDownloadEvent.getVerCode(),
         editorialDownloadEvent.getPath(), editorialDownloadEvent.getPathAlt(),
-        editorialDownloadEvent.getObb(), false, editorialDownloadEvent.getSize()))
+        editorialDownloadEvent.getObb(), false, editorialDownloadEvent.getSize(), null, null))
```

Comment:
This is assuming that there will be no appbundles in editorial, right?

Reply:
you are assuming wrong! it is there already